frontend/src/components/partials/SingleIdeaContent/CommentsSection.tsx

```diff
@@ -101,9 +104,13 @@ if(ideaComments){
         ) : (
           ideaComments &&
           ideaComments.map((comment) => (
+            <div className="textlimit & textlimit:hover">
```

Comment:
What's the point of the :hover portion of the classname? Do comments have some sort of hover functionality?

Also, just curious: what does the & do here? I've never seen this before and I can't find anything about it online

Reply:
That was something I forgot to delete and I've updated it. Thanks for catching it!!